tox.ini

```diff
@@ -16,10 +16,11 @@ basepython =
     pypy3: pypy3
     py37: python3.7
     py38: python3.8
-    # See https://github.com/tox-dev/tox/issues/1548
-    {check,reformat,docs,build}: python3
+    docs: python3.8
```

Comment:
Despite this, the `docs` environment keeps insisting on Python 3.9 on CI https://dev.azure.com/poliastro/poliastro/_build/results?buildId=561&view=logs&j=f7cd7807-82c0-5095-6e59-d04f697ea1d8&t=6b55555c-5bc2-5302-9933-bfad45886ef1&l=128 why??

Reply:
Oh, it's not about `docs`, it's about `build`.

Reply:
Left a comment https://github.com/tox-dev/tox/issues/1330#issuecomment-723932826

Reply:
And another one https://github.com/tox-dev/tox/issues/1738